src/traces/scatter/defaults.js

```diff
@@ -76,7 +76,7 @@ module.exports = function supplyDefaults(traceIn, traceOut, defaultColor, layout
         dfltHoverOn.push('fills');
     }
     coerce('hoveron', dfltHoverOn.join('+') || 'points');
-    if(traceOut.hoveron !== 'fills') coerce('hovertemplate');
+    coerce('hovertemplate', false);
```

Comment:
We could oossibly use
```suggestion
    coerce('hovertemplate');
```
so that `hovertemplate` defaults to blank string.

Reply:
Could we keep the initial if statement and just add a condition for *lines* `mode`?
```js
if(traceOut.hoveron !== 'fills' || traceOut.mode === 'lines') coerce('hovertemplate');
```

Reply:
I have updated the code to use this variant instead. Not sure why I used false as the default value